packages/tools/client-debugger/client-debugger/src/FluidClientDebugger.ts

```diff
@@ -161,8 +209,12 @@ export class FluidClientDebugger
 	 */
 	public dispose(): void {
 		// Unbind Container events
+		this.container.off("attached", this.containerAttachedHandler);
 		this.container.off("connected", this.containerConnectedHandler);
+		this.container.off("contextChanged", this.containerContextChangedHandler);
 		this.container.off("disconnected", this.containerDisconnectedHandler);
+		this.container.off("disposed", this.containerDisposeddHandler);
```

Comment:
I actually wasn't aware that IContainer had a distinction between `closed` and `disposed` states. Our UI currently treats these two states as identical. Once we get to the UI side of this work, it would probably be worth updating our views to distinguish correctly.

Reply:
`closed` and `disposed` all meaning this container permanently disabled. Then since `IContainer` identifies these them as two different states, should we keep these two states?

Reply:
Closed is one step before Disposed, or at least that's my understanding from reading the docs. Closed indicates that no new activity will occur, but resources have not necessarily been disposed yet. Disposed indicates Closed + resources have been disposed. I think differentiating those in the visualization would be good, since they are two different states we expose on the API surface. It will also be a good opportunity to educate users to the difference between the two.

Reply:
I see! Thanks for clarifying. I will update the comment about them.